model_compression_toolkit/target_platform_capabilities/schema/schema_functions.py

```diff
@@ -82,7 +82,10 @@ def is_opset_in_model(tp_model: TargetPlatformModel, opset_name: str) -> bool:
         bool: True if an OperatorsSet with the given name exists in the target platform model,
               otherwise False.
     """
-    return opset_name in [x.name for x in tp_model.operator_set]
```

Comment:
`return tp_model.operator_set is not None and opset_name in [x.name for x in tp_model.operator_set]`